### DISCUSSION TITLE
죄송합니다만 북토크 일정을 조율할 수 있을까요?

### DISCUSSION THREAD

**Discussion by yoonucho** [ThinkAboutSoftware/AcademicConference#28] [General]
안녕하세요 
다름이 아니라 제가 바쁜 회사일정때문에 이번주 금요일 북토크가 도무지 아무래도 힘들거  같습니다. ㅠㅠ
Hoxy  다음주 화요일 18일은 어떠신지요 

정말 죄송합니다 😿

**Comment by jongfeel** [comment]
저는 됩니다. 다른 분들은 어떨지 모르겠네요

**Comment by hi-hyein** [comment]
저도 가능합니다!

**Reply by Joseph-Cha** [reply]
저도 당근 가능합니당!!

**Comment by jongfeel** [comment]
그러면 일정을 다음 주 화요일로 변경하겠습니다.
다음날이 휴일이라 부담 없을 수도 있겠네요.

뜻밖에 시간을 많이 확보했으니, 코멘트를 제대로 확인하고 달아볼 수 있을 것 같습니다.

**Comment by jongfeel** [comment]
@yoonucho @hi-hyein @Joseph-Cha 
한가지 추가해 드리면 클린코더 다음 책 읽는 시간은 바뀌지 않고 그대로입니다.

즉, 5월 15일 부터는 5월 18일날 진행하는 리뷰와 관계 없이 다음 챕터 읽는 건 진행해 주셔야 해요!

**Reply by yoonucho** [reply]
네 알고 있습니다 !

**Comment by yoonucho** [comment]
정말 죄송하고 감사드려요 ㅠ
저 내일도 야근각이라 토요일까지 리뷰올릴게요
